keyboard/src/data/keyboard_raw.ycp

```diff
@@ -534,7 +534,7 @@ return ($[
 	_("Ukrainian"),
 	$[
 	    "pc104"	: $[
-		    "ncurses"	: "ua.map.gz",
+		    "ncurses"	: "ua-utf.map.gz",
```

Comment:
The original intention was to include this fix as part of https://github.com/yast/yast-country/pull/61 But since that pull request has been unfortunately blocked for quite some time (I expect to spend time on it today to unblock it), I will merge your pull request today and rebase pr#61 on top.